src/app/pages/deposits/no-deposits/no-deposits.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-no-deposits',
+  templateUrl: './no-deposits.component.html',
+  styleUrls: ['./no-deposits.component.scss']
+})
+export class NoDepositsComponent implements OnInit {
```

Comment:
Also please cross check if this component was used anywhere.

Reply:
Thanks @CryceTruly , I have worked on removing this component as it was indeed useless